tests/unit/expire.tcl

```diff
@@ -285,22 +281,22 @@ start_server {tags {"expire"}} {
         r getex foo4 pxat [expr [clock milliseconds]+10000]
         assert_replication_stream $repl {
             {select *}
-            {set foo1 bar PX 200000}
-            {set foo1 bar PX 100000}
             {set foo1 bar PXAT *}
             {set foo1 bar PXAT *}
-            {set foo1 bar PX 100000}
-            {set foo1 bar PX 100000}
+            {set foo1 bar PXAT *}
```

Comment:
Since we're deleting all of the validation that the two values are the same (Just that we're replicating it as a PEXIREAT or SET PXAT). Can we also test that on the primary and replica executing TTL gives the same value?

Reply:
Makes sense to not reduce the test coverage. Done.

Had to Introduce a new `[ABSOLUTE]` optional argument to `TTL` and `PTTL` commands to return absolute expiration timestamps of keys.

Reply:
I ended up introducing a new `EXPIRETIME` command to query absolute timestamps of TTLs.